BusyTiming/HomePage/TimerVM.swift

```diff
@@ -24,7 +24,7 @@ class TimerVM {
     var cancellable: AnyCancellable?
 
     init() {
-        // **** tale the timet values
+        // **** tale the timer values
```

Comment:
I've just added such comments (starting with '*****') to show the idea. Now they all can be removed.